lib/LabeledTextItem/LabeledTextItem.less

```diff
@@ -10,7 +10,6 @@
 .moon-labeledtextitem .label {
 	.moon-sub-header-text;
 	color: @moon-label-text-color;
-	border-top: 3px solid @moon-label-border-color;
 	margin: 0px 0px 3px 0px;
 	padding: 6px 0px 0px 0px;
```

Comment:
I think only the `border-top` and `height` were mentioned in the ticket, based on discussion with Gruettner. @Djspaceg, do you have any thoughts on this?